tests/testmongodb.nim.cfg

```diff
@@ -0,0 +1 @@
+-d:nimOldCaseObjects
```

Comment:
Can we switch to the new case objects and eliminate this switch?

Reply:
Also, can we use .nims for configuration?

Reply:
I sent a message to the `nimongo` repo requesting the few simple changes needed to make `nimongo` ready for nim 0.20. Not heard back yet.

Reply:
Sent another message and included another developer. Hopefully something happens soon.

Needing to use the the switch is silly given how easy it is to use the new case object methods. Plus nim 1.0 is coming soon and it would be good to have `nimongo` ready for it.